libs/accounts/src/lib/transfer-form.tsx

```diff
@@ -72,8 +72,26 @@ export const TransferForm = ({
 
   const assets = sortBy(
     accounts
-      .filter((a) => a.type === AccountType.ACCOUNT_TYPE_GENERAL)
+      .filter(
+        (a) =>
+          a.type === AccountType.ACCOUNT_TYPE_GENERAL ||
+          a.type === AccountType.ACCOUNT_TYPE_VESTED_REWARDS
+      )
+      .reduce((merged, account) => {
```

Comment:
This doesn't fix the issue. I still get the same general balance. Taking a look at this now...

Reply:
Just to clarify, this does fix _a_ issue where the balance in the asset dropdown was only the general account. But not the one where the balance in the to account dropdown is incorrect